docs/io/visualization/generating_widgets.ipynb

```diff
@@ -179,11 +185,58 @@
     "<div class=\"alert alert-info\">\n",
     "\n",
     "Note\n",
-    "    \n",
+    "\n",
     "The virtual packet logging capability must be active in order to produce virtual packets' spectrum in `Line Info Widget`. Thus, make sure to set `virtual_packet_logging: True` in your configuration file. It should be added under `virtual` property of `spectrum` property, as described in [configuration schema](https://tardis-sn.github.io/tardis/using/components/configuration/configuration.html#spectrum).\n",
     "\n",
-    "</div>"
+    "</div>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Energy Level Widget\n",
+    "\n",
+    "This widget lets you explore the visualize the various line interactions in a simulation in the form of an Energy Level Diagram.\n",
```

Comment:
This widget lets you explore and visualise the various level populations and line interactions in a simulation in the for of an Energy Level Diagram.